scripts/install-helpers/install.sh

```diff
@@ -219,6 +219,31 @@ function override_trustee_image() {
     fi
 }
 
+# Function to workaround an operator issue, as the operator does not
+# properly take the cluster wide proxy set into consideration.
+function ensure_proxy_is_set_workaround() {
+    CLUSTER_HTTPS_PROXY="$(oc get proxy/cluster -o jsonpath={.spec.httpsProxy})"
+    CLUSTER_HTTP_PROXY="$(oc get proxy/cluster -o jsonpath={.spec.httpProxy})"
+    CLUSTER_NO_PROXY="$(oc get proxy/cluster -o jsonpath={.spec.noProxy})"
+
+    # Here were coming with the assumption that the env is always not there, which is
+    # the case in a normal deployment.
+    #
+    # ideally, we should double check whether it's empty, and yatta yatta ... but if
+    # this is just a workaround till the Operator knows how to properly do this.
+    oc patch -n trustee-operator-system deployment trustee-deployment --type=json -p="[
```

Comment:
can we do the patch only if the proxy are defined cluster-wide? This is what we get in the deployment:
```
        env:
        - name: HTTPS_PROXY
        - name: HTTP_PROXY
        - name: NO_PROXY
```

Reply:
I thought about that, but there's no harm in also doing the patch if proxy is *not* set cluster-wide (I've tested this to ensure no issues would be found).

It increases the logic if we want to take that approach, as then we'd have to check for each one of env vars, and patch accordingly depending on which one was set.

IMHO, for this effort, it's not worth the trouble.